app/forms/publish/accept_terms_form.rb

```diff
@@ -0,0 +1,23 @@
+module Publish
+  class AcceptTermsForm < BaseModelForm
```

Comment:
I think we should namespace this a bit more

```suggestion
module Interruption
  class AcceptTermsForm < BaseModelForm
```

Reply:
done